### ISSUE TITLE
Unable to Access via IP Camera Viewer (iOS) and browser via WAN

### ISSUE THREAD

**Issue by tripchoc** [EliasKotlyar/Xiaomi-Dafang-Hacks#672]
I am able to access RTSP via VLC, browser, IP Camera Viewer (iOS) on LAN but could not connect via WAN. 
I have tried to set server.port = xx and enable port forwarding on my router (Apple Airport Extreme) and assigned a static IP to the Dafang camera. (the port forwarding work correctly for my existing DLink IP Cam). 

My username and password for HTTPS and RTSP have been set to the same. 
RTSP has been changed to yy and viewing works perfectly in LAN but I have changed back to default 8554. 

Basically just want to view multiple viewing streams on iOS devices (does not own an Android to run TinyCam). Am I missing something? I tried to do a search but could not find anything..

**Comment by jmtatsch** [comment]
Which IP Camera Viewer App exactly?

**Comment by tripchoc** [comment]
This is the IP Cam Viewer app, it has Xiaomi Dafang Hack profile in there
https://itunes.apple.com/sg/app/ip-cam-viewer-pro/id402656416?mt=8

**Comment by tripchoc** [comment]
Please close this case. It is working now. Thank you.

**Comment by jmtatsch** [comment]
What was your mistake?

**Comment by tripchoc** [comment]
On lighttpd.conf - did a change on server.port = 80 to server.port=’1234’; did a change on the SSL port from 443 to '456' and maintained the RTSP port as default (8554). 

$SERVER["socket"] == ":456" {
 ssl.engine = "enable"
 ssl.pemfile = "/system/sdcard/config/lighttpd.pem"
 }

Configured port forwarding ‘1234’, '456', '8554' and point it to the assigned static LAN IP of the camera on the router. 
Managed to gain access to the camera from WAN, on browser, access by https://dyndns:456 with username and password. 
For IP Cam Viewer (iOS app), login with the username and password with SSL enabled and using '456' as input. 

Below function does not work anymore. 
Could not login through http://dyndns:1234 (Perhaps due to the re-direction to SSL?) 
Tried to view on VLC (rtsp://LAN IP:8554/unicast), keyed in the username and password and it does not allow me to go through anymore, even though it was keyed correctly. (not sure if its due to re-direction)

Lastly, I could not get audio up. Not sure why as it used to work for me. 
The only option of logging in to view is through SSL only. 

Below are my audio config in rtspserver.conf 

AUDIOFORMAT=PCMU
AUDIOINBR=8000
AUDIOOUTBR=8000
HWVOLUME=120
SWVOLUME=-1
FILTER=1
HIGHPASSFILTER=false
AECFILTER=false

**Comment by donoussa** [comment]
Hello, 
I have the same problem... cant view outside LAN. 
Dafang hack works OK. 
I can view camera from LAN. 
I changed Port 80 to 456.
RTSP port 8554 left as is. 
I have done port forwarding on the router but nothing.  456 -- CameraLocalIP - 456. 
Android App - Tinycam Pro.

Can you please help . 
Thank you in Advance.
